receiver/sapnetweaverreceiver/metadata.yaml

```diff
@@ -9,9 +9,16 @@ resource_attributes:
     description: The SAP Netweaver node.
 
 attributes:
+  response_type:
+    type: string
+    description: The response type
+    enum:
+    - transaction
+    - dialog
+    - dialogRFC
+    - http
   control_state:
     type: string
-    value: state
```

Comment:
Why did we get rid of this? I feel like having the value be `state` is a tad more concise/usable?

Reply:
So In the Makefile, I get the latest [mdatagen](https://github.com/observIQ/observiq-otel-collector/blob/main/Makefile#L95) in which the `value` field got [deprecated.](https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/c6977cad7fabc67a8129aa0b67be4fbfcf6d0aad)  . There is now a `NameOverride` that can be used which I can put in.

Reply:
Okay that sounds good to switch over to that 👍